akka-cluster-tools/src/multi-jvm/scala/akka/cluster/client/ClusterClientSpec.scala

```diff
@@ -37,9 +32,13 @@ object ClusterClientSpec extends MultiNodeConfig {
     akka.remote.log-remote-lifecycle-events = off
     akka.cluster.auto-down-unreachable-after = 0s
     akka.cluster.client.heartbeat-interval = 1s
-    akka.cluster.client.acceptable-heartbeat-pause = 3s
+    akka.cluster.client.acceptable-heartbeat-pause = 1s
```

Comment:
perhaps these new tests deserve a new `Spec` to not interfere with existing tests